QUARKUS-2394.md

```diff
@@ -0,0 +1,39 @@
+# QUARKUS-2394 Drop OpenTelemetry Jaeger exporter
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2394
+
+Opentelemetry Jaeger exported has been deprecated from an official Quarkus support point of view. The extension `quarkus-opentelemetry-exporter-jaeger` will be moved to Quarkiverse and `quarkus-opentelemetry-exporter-otlp` extension has been promoted as the supported one. 
+
+## Scope of the testing
+
+All references to `quarkus-opentelemetry-exporter-jaeger` are going to be removed. Current coverage are going to be replaced by `quarkus-opentelemetry-exporter-otlp` on OpenShift / Baremetal / Native
+
```

Comment:
Add step to ensure metadata for TP are dropped

- update Marete ... https://gitlab.cee.redhat.com/rhoar-qe/marete/-/blob/main/src/main/resources/quarkus-2-13.yml#L282
  - that update needs to be done after the metadata changes are in
  - push on the JIRA to have the metadata update done
- you should check code.quarkus stage instance related to ER/CR build to visually ensure the TP label is gone
  - (after the metadata stuff is done and we have prod build with it)

Reply:
Done: https://gitlab.cee.redhat.com/rhoar-qe/marete/-/merge_requests/241